Source/Utils/WarpXUtil.cpp

```diff
@@ -247,9 +247,19 @@ Parser makeParser (std::string const& parse_function, amrex::Vector<std::string>
 
     Parser parser(parse_function);
     parser.registerVariables(varnames);
-    ParmParse pp_my_constants("my_constants");
+
     std::set<std::string> symbols = parser.symbols();
     for (auto const& v : varnames) symbols.erase(v.c_str());
+
+    // PICMI specifies inputs under this name, through which expressions
```

Comment:
Note that this capability is not PICMI specific but also applies to plain C++ input files as well.

Reply:
```suggestion
    // User can provide inputs under this name, through which expressions
```